docs/source/tutorials/improving_ml_performance.ipynb

```diff
@@ -0,0 +1,3574 @@
+       "              multi_strategy=None, n_estimators=None, n_jobs=None,\n",
+       "              num_parallel_tree=None, objective=&#x27;multi:softprob&#x27;, ...)</pre></div> </div></div></div></div>"
+      ],
+      "text/plain": [
+       "XGBClassifier(base_score=None, booster=None, callbacks=None,\n",
+       "              colsample_bylevel=None, colsample_bynode=None,\n",
+       "              colsample_bytree=None, device=None, early_stopping_rounds=None,\n",
+       "              enable_categorical=True, eval_metric=None, feature_types=None,\n",
+       "              gamma=None, grow_policy=None, importance_type=None,\n",
+       "              interaction_constraints=None, learning_rate=None, max_bin=None,\n",
+       "              max_cat_threshold=None, max_cat_to_onehot=None,\n",
+       "              max_delta_step=None, max_depth=None, max_leaves=None,\n",
+       "              min_child_weight=None, missing=nan, monotone_constraints=None,\n",
+       "              multi_strategy=None, n_estimators=None, n_jobs=None,\n",
+       "              num_parallel_tree=None, objective='multi:softprob', ...)"
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "train_labels = train_labels[\"noisy_letter_grade\"]\n",
+    "clf = XGBClassifier(tree_method=\"hist\", enable_categorical=True, random_state=SEED)\n",
+    "clf.fit(train_features, train_labels)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "944efa9e-dec4-474f-8007-73587453f8a6",
+   "metadata": {},
+   "source": [
+    "**In a Jupyter environment, please rerun this cell to show the HTML representation or trust the notebook.\n",
+    "On GitHub, the HTML representation is unable to render, please try loading this page with nbviewer.org.**\n",
+    "\n",
+    "Although curating clean test data does not directly help train a better ML model, more reliable model evaluation can improve our overall ML project. For instance, clean test data can enable better informed decisions regarding when to deploy a model and better model/hyperparameter selection."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c214e30b-4c82-4295-a3b0-68493904836b",
+   "metadata": {},
+   "source": [
+    "## 4. Compute out-of-sample predicted probabilities for test data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "id": "044c0eb1-299a-4851-b1bf-268d5bce56c1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.model_selection import cross_val_predict\n",
+    "\n",
+    "test_labels = test_labels[\"noisy_letter_grade\"].astype(int)\n",
+    "\n",
+    "num_crossval_folds = 5\n",
+    "test_pred_probs = cross_val_predict(\n",
+    "    clf,\n",
+    "    test_features,\n",
+    "    test_labels,\n",
+    "    cv=num_crossval_folds,\n",
+    "    method=\"predict_proba\",\n",
+    ")"
```

Comment:
```suggestion
    "test_pred_probs = clf.predict_proba(test_features)"
```

Reply:
@mturk24 please re-run the entire tutorial with this change and verify results are still good. 

We just want to use the already-trained model to get the held-out test data predictions